src/lib/hooks/useFilteredFields.ts

```diff
@@ -1,20 +1,26 @@
 import { useEffect } from 'react';
-import { EProjectFields } from '../consts/projects';
+import {
+  IActiveFilters,
+  IWatchFields,
+} from '../components/commons/FilterPanel';
 
-export const useFilteredFields = (
-  watchedFields: any,
-  setActiveFilters: any,
+const getEntries = <T extends {}>(obj: T) =>
```

Comment:
this function should be in a utils directory... probably arrayUtils file or something like this

Reply:
good idea! moving it to arrayUtils